test/unit/source.js

```diff
@@ -283,38 +306,39 @@ describe('Sources', function () {
             assert.throws(() => new FileSource({ crs: 'EPSG:4326' }), Error);
         });
 
-        describe('should set the crs projection from features', function () {
-            it('with the crs', function () {
-                const source = new FileSource({
-                    features: { crs: 'EPSG:4326' },
-                });
-                assert.strictEqual(source.crs, 'EPSG:4326');
-            });
-
-            it('with the crs projection', function () {
-                const source = new FileSource({
-                    features: { crs: 'EPSG:4326' },
-                });
-                assert.strictEqual(source.crs, 'EPSG:4326');
+        it('should set the crs projection from features', function () {
+            const source = new FileSource({
+                features: { crs: 'EPSG:4326' },
+                format: 'application/json',
             });
+            assert.strictEqual(source.crs, 'EPSG:4326');
         });
     });
 
     describe('C3DTilesSource', function () {
-        const params3DTiles = {
-            url: 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/3DTiles/lyon_1_4978/tileset.json',
-            networkOptions: process.env.HTTPS_PROXY ? { agent: new HttpsProxyAgent(process.env.HTTPS_PROXY) } : {},
-        };
+        let stubFetcherJson;
+        before(function () {
+            stubFetcherJson = sinon.stub(Fetcher, 'json')
```

Comment:
The purpose of this PR is to get in the direction of real unti tests. Thus i did mock the fetching everywhere, but later on we should be mocking the process that use the mocked fetcher (except for some specifique tests).That's why I think that a it will be on some further PR to think about how to factorize the different mocking parts.

Reply:
Ok, in this case could you open an issue about it please ?